buildbuddy.yaml

```diff
@@ -9,7 +9,7 @@ actions:
         branches:
           - "main"
     bazel_commands:
-      - "test --config=workflows //tools/... //test/..."
+      - "test --config=workflows //tools/... //test/... //test/integration:all_integration_tests"
```

Comment:
This shouldn't be needed, right? Doesn't `//test/...` cover it?

Reply:
Right now, I the tests are configured as `manual` and `exclusive`. This can be changed.  I keep them `manual` for my projects so that `bazel test //...` remains responsive. The integration tests can take some time to execute. When you multiply that with the different versions of Bazel, it may be best to execute them when requested. I am happy to make them execute with other tests.  Just let me know.

Reply:
No, that makes sense. Thanks!